src/titiler/core/titiler/core/templates/map.html

```diff
@@ -1,146 +1,182 @@
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>TiTiler Map Viewer</title>
+    <meta
+      name="viewport"
+      content="initial-scale=1,maximum-scale=1,user-scalable=no"
+    />
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/leaflet@1.9.3/dist/leaflet.css"
+      integrity="sha384-o/2yZuJZWGJ4s/adjxVW71R+EO/LyCwdQfP5UWSgX/w87iiTXuvDZaejd3TsN7mf"
+      crossorigin="anonymous"
+    />
+    <script
+      src="https://unpkg.com/leaflet@1.9.3/dist/leaflet.js"
+      integrity="sha384-okbbMvvx/qfQkmiQKfd5VifbKZ/W8p1qIsWvE1ROPUfHWsDcC8/BnHohF7vPg2T6"
+      crossorigin="anonymous"
+    ></script>
+    <script
+      src="https://unpkg.com/proj4@2.3.14/dist/proj4.js"
+      integrity="sha384-R7x++v2MKcATI+D1/GJsn636xbHca492Sdpm8BD36lj5vdWB9+OUBpM1oKkrzqv9"
+      crossorigin="anonymous"
+    ></script>
+    <script
+      src="https://unpkg.com/proj4leaflet@1.0.2/src/proj4leaflet.js"
+      integrity="sha384-aDnBHDK9AhLbrYhThBxEVMriFbix8Sz2059IlD3HbZhz7+WNmz+pSkOcI7MY72cE"
+      crossorigin="anonymous"
+    ></script>
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+        width: 100%;
+        height: 100%;
+        background-color: #e5e5e5;
+      }
+      #map {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        width: 100%;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="map"></div>
 
     <script type="text/javascript">
 
-const bboxPolygon = (bounds) => {
-    var LL_EPSILON = 1e-6
-    return {
-        'type': 'Feature',
-        'geometry': {
-            'type': 'Polygon',
-            'coordinates': [[
-                [bounds[0] + LL_EPSILON, bounds[1] + LL_EPSILON],
-                [bounds[2] -  LL_EPSILON, bounds[1] + LL_EPSILON],
-                [bounds[2] - LL_EPSILON, bounds[3] - LL_EPSILON],
-                [bounds[0] + LL_EPSILON, bounds[3] - LL_EPSILON],
-                [bounds[0] + LL_EPSILON, bounds[1] + LL_EPSILON]
-            ]]
-        },
-        'properties': {}
-    }
-}
-
-var crs = new L.Proj.CRS(
-    '{{ tms.crs.srs }}',
-    '{{ tms.crs.to_proj4() }}', {
-    origin: [{{ tms.xy_bbox.left }}, {{ tms.xy_bbox.top }}],
-    bounds: L.bounds(
-      L.Point({{ tms.xy_bbox.left}}, {{ tms.xy_bbox.bottom }}),
-      L.Point({{ tms.xy_bbox.right}}, {{ tms.xy_bbox.top }})
-    ),
-    resolutions: {{ resolutions|safe }},
-  }
-);
-
-
-var map = L.map('map', {
-  crs: crs,
-  minZoom: {{ tms.minzoom }},
-  maxZoom: {{ tms.maxzoom }},
-  attributionControl: false
-});
-
-L.control.attribution({prefix: '<a href="https://leafletjs.com" target="_blank">Leaflet</a> | <a href="https://developmentseed.org/titiler" target="_blank">Titiler</a>'}).addTo(map)
-
-const nullIsland = L.marker([0, 0]).addTo(map);
-const madrid = L.marker([40, -3]).addTo(map);
-const london = L.marker([51.50722, -0.1275]).addTo(map)
-const auckland = L.marker([-36.864664, 174.792059]).addTo(map);
-const seattle = L.marker([47.596842, -122.333087]).addTo(map);
```

Comment:
I'm fine removing those marker. I've used them to control the map was correct when using custom TMS. Maybe we should find something better like a really light geojson

Reply:
Something like a geojson around the Blagden Alley building?

Reply:
Or the Bermuda Triangle?

Reply:
yeah that would be nice!

Reply:
Actually, I'll just keep the familiar point locations but add a keyboard shortcut (and/or a tiny button) to toggle them on/off.

Reply:
How does this look?
![image](https://github.com/user-attachments/assets/3b835f5b-547a-4766-bf59-91b3c2d65ae3)

I added a keyboard shortcut (`b`) for toggling the original placemarks on and off, and a tiny button at the bottom right. They are not displayed by default but are easy to toggle on.